prow/config/config.go

```diff
@@ -2179,7 +2193,6 @@ func (c Config) validatePostsubmits(postsubmits []Postsubmit) error {
 
 // validatePeriodics validates a set of periodics.
 func (c Config) validatePeriodics(periodics []Periodic) error {
-
```

Comment:
Let's not introduce diffs like these that are not related to the code that's being changed.

Reply:
Removed 👍